test/data-source.test.js

```diff
@@ -60,8 +60,8 @@ describe('DataSource', function() {
       });
 
       var Color = ds.createModel('color', {name: String});
-      assert(Color.prototype instanceof loopback.Model);
-      assert.equal(Color.base, loopback.Model);
+      assert(Color.prototype instanceof Color.registry.getModel('Model'));
+      assert.equal(Color.base.modelName, 'PersistedModel');
```

Comment:
I'd prefer to keep constructor-equality checks here instead of switching to name-based checks. Is there any particular reason why it's not enough to replace `loopback.` with `Color.registry.` and keep the rest unchanged?